package protoFile.login.enterInstance;
message enterInstanceRequest{
     required int32 id = 1;//角色Id
     required int32 InstanceId = 2;//副本Id
}

message enterInstanceResponse{
     required bool result = 1;
     optional string message = 2;
     optional ResponseData data=3;
}

message ResponseData{
     optional int32 placeId = 1;//场景Id
}
